.github/workflows/ci.yaml

```diff
@@ -0,0 +1,98 @@
+name: CI
+
+on: [push, pull_request]
+
+jobs:
+  test:
+    runs-on: ${{ matrix.os }}
+    continue-on-error: ${{ matrix.experimental }}
+    strategy:
+      fail-fast: true
+      matrix:
+        os: ["windows-latest", "ubuntu-latest", "macos-latest"]
+        python-version: ["3.7", "3.8"]
+        experimental: [false]
+        include:
+          - python-version: "3.8"
+            os: "ubuntu-latest"
+            experimental: true
+
+    env:
+      PYTHON_VERSION: ${{ matrix.python-version }}
+      OS: ${{ matrix.os }}
+      UNSTABLE: ${{ matrix.experimental }}
+      ACTIONS_ALLOW_UNSECURE_COMMANDS: true
+
+    steps:
+      - name: Checkout source
+        uses: actions/checkout@v2
+
+      - name: Setup Conda Environment
+        uses: conda-incubator/setup-miniconda@v2
+        with:
+          miniconda-version: "latest"
+          python-version: ${{ matrix.python-version }}
+          mamba-version: "*"
+          channels: conda-forge,defaults
+          environment-file: continuous_integration/environment.yaml
+          activate-environment: test-environment
+
+      - name: Install unstable dependencies
+        if: matrix.experimental == true
+        shell: bash -l {0}
+        run: |
+          python -m pip install \
```

Comment:
how about pyproj for unstable dependencies?

Reply:
Good call. I'll add that later today and see if I can get test pypi working after a merge.

Reply:
Done, but it currently fails because it needs proj>=7.2 and conda-forge is only installing 7.0 for now. I've commented on the pyproj conda-forge package repository since it doesn't seem to be requiring this newer version. I've also updated this job in my latest commit to try to bump the version of proj and see if that lets the pip install of pyproj succeed.